modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/SCIM2GroupTestCase.java

```diff
@@ -331,4 +317,33 @@ private String getUsersPath() {
     private String getAuthzHeader() {
         return "Basic " + Base64.encodeBase64String((adminUsername + ":" + password).getBytes()).trim();
     }
+
+    private void validateFilteredGroup(String attributeName, String operator, String searchAttribute) throws IOException {
+
+        String userResourcePath = getPath() + "?filter=" + attributeName + operator +
+                searchAttribute;
```

Comment:
formatting issue here. we can use searchAttribute also in the above line.

Reply:
fixed